pkg/genericapiserver/server/config.go

```diff
@@ -169,6 +169,9 @@ type ServingInfo struct {
 	// BindNetwork is the type of network to bind to - defaults to "tcp", accepts "tcp",
 	// "tcp4", and "tcp6".
 	BindNetwork string
+
+	// ipBasedLimit determines the maximum number of concurrent requests based on ip
```

Comment:
everything in this struct should be exported. this config is what allows driving the generic api server from consuming components